### PR TITLE
Dl 971b

### PR DESCRIPTION
**Pull request by robert-webb-hmrc**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -373,7 +376,7 @@ pla.exitSurvey.question2                                = <p class="bold-small">
 pla.exitSurvey.question3                                = <p class="bold-small">Pa mor debygol ydych o argymell eich cyfrif treth i''ch ffrindiau a''ch teulu?</p>
 pla.exitSurvey.question4                                = <p class="bold-small">Yn gyffredinol, sut ydych yn teimlo am eich profiad wrth ddefnyddio''ch cyfrif treth heddiw?</p>
 pla.exitSurvey.sendFeedback                             = Anfon adborth
-pla.exitSurvey.feedback.optional                        = Rydym yn defnyddio’ch adborth i wella ein gwasanaethau.
+pla.exitSurvey.feedback.optional                        = Rydym yn defnyddio''ch adborth i wella ein gwasanaethau.
```
Comment (paulanderson1234) [inline conf/messages.cy]: Can't find this being utilised anywhere on any view and no equivelent in English message file using same key

File: conf/messages.cy [hunk 2]
```diff
@@ -373,7 +376,7 @@ pla.exitSurvey.question2                                = <p class="bold-small">
 pla.exitSurvey.question3                                = <p class="bold-small">Pa mor debygol ydych o argymell eich cyfrif treth i''ch ffrindiau a''ch teulu?</p>
```
Comment (paulanderson1234) [inline conf/messages.cy]: Should we be adding styles etc to the message files - may need some investigation on subsequent story

File: conf/messages.cy [hunk 3]
```diff
@@ -333,7 +336,7 @@ pla.resultSuccess.viewDetails           = Gallwch <a href="{0}">fwrw golwg ar fa
 pla.resultSuccess.viewDetailsLinkText   = fwrw golwg ar fanylion eich diogelwch, neu eu newid
```
Comment (paulanderson1234) [inline conf/messages.cy]: Cannot fin equivalent key in English messages - should there be?

File: conf/messages.cy (hunk 2 above)
Comment (Tom-Stacey) [reply conf/messages.cy]: We definitely shouldn't be

Comment (Tom-Stacey) [reply conf/messages.cy]: Sometimes it can't be avoided with the way a service has been coded, but we should definitely remove if we can

**Pull request merged**